### DISCUSSION TITLE
Location of user picture in Win 10 for the Win 7 menu

### DISCUSSION THREAD

**Discussion by gr38w** [Open-Shell/Open-Shell-Menu#1091] [Q&A]
I'm trying to locate the image displayed at the top of the right hand side in the Win-7 menu. I'm running Win-10. Can anyone help me?

**Comment by CTVCAM8** [comment]
A screenshot would have been helpful.

1 If you are talking about when Windows 10 is first installed with the wire frame head & shoulders:

![user-48](https://user-images.githubusercontent.com/44127216/184511279-f4c846bb-0a3b-4772-ba19-0342d4b55347.png)


Make sure to check the "Hidden items" box in the "View" tab of Explorer.

C:\ProgramData\Microsoft\User Account Pictures
or
C:\ProgramData\Microsoft\Default Account Pictures

2 If you set up a different picture but forgot how to do it:
Go to Settings, Accounts, Browse for one

3  If you set up the sharper internal .bmp or.png user picture option in Open-Shell (Does not use Windows user picture),
it's in Open-Shell settings, "General Behavior" tab, "User picture".

If you are talking about changing the icons that show in the user picture location of the 7 style start menu,
like "Pictures", "Documents", Use the "Customize Start Menu" tab to edit the item with a different icon location.

**Comment by widapi382-cloud** [comment]
what aboutn changing the picture in the colums for start menu?
